class Solution {
    public int lcm(int a, int b) {
      
      int x=a;
      int y=b;
      while(y!=0){
          int temp=y;
          y=x%y;
          x=temp;
      }
      return (a/x)*b;
    }
} 
  
